pkg/gui/services/custom_commands/session_state_loader.go

```diff
@@ -19,11 +20,47 @@ func NewSessionStateLoader(c *helpers.HelperCommon, refsHelper *helpers.RefsHelp
 	}
 }
 
+type Commit struct {
+	Hash          string
+	Sha           string
+	Name          string
+	Status        models.CommitStatus
+	Action        todo.TodoCommand
+	Tags          []string
+	ExtraInfo     string
+	AuthorName    string
+	AuthorEmail   string
+	UnixTimestamp int64
+	Divergence    models.Divergence
+	Parents       []string
```

Comment:
Some of these are probably not useful in custom commands, but they were exposed before, so we have to keep supporting them I suppose.